rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/MonitoringHostView.java

```diff
@@ -210,7 +210,8 @@ public void close() {
     @Override
     public void onFailure(Throwable caught) {
         close();
-        status.setContents(JSONUtils.getJsonErrorMessage(caught));
+        status.setContents("Monitoring is unavailable on this host. Maybe node was started with --disableMonitoring option.<br>"+
```

Comment:
Monitoring is unavailable on this host. Either monitoring was disabled with the disableMonitoring option or an error occurred. More information are available inside the node logs.